metricbeat/module/vsphere/host/_meta/fields.yml

```diff
@@ -20,22 +16,120 @@
       type: long
       description: >
         Free CPU in Mhz
-    - name: memory.used.bytes
+    - name: disk.capacity.usage.bytes
       type: long
       description: >
-        Used Memory in bytes
+        The amount of storage capacity currently being consumed by or on the entity.
+    - name: disk.devicelatency.average.ms
+      type: long
+      description: >
+        Average amount of time it takes to complete an SCSI command from physical device.
+    - name: disk.latency.total.ms
+      type: long
+      description: >
+        Highest latency value across all disks used by the host.
+    - name: disk.read.bytes
+      type: long
+      description: >
+        Average number of bytes read from the disk each second.
       format: bytes
-    - name: memory.total.bytes
+    - name: disk.write.bytes
       type: long
       description: >
-        Total Memory in bytes
+        Average number of bytes written to the disk each second.
+      format: bytes
+    - name: disk.total.bytes
+      type: long
+      description: >
+        Aggregated disk I/O rate.
       format: bytes
     - name: memory.free.bytes
       type: long
       description: >
         Free Memory in bytes
       format: bytes
+    - name: memory.total.bytes
+      type: long
+      description: >
+        Total Memory in bytes
```

Comment:
Can we put `.` at the end of each descriptions?

Reply:
Sure